index.js

```diff
@@ -134,7 +162,10 @@ function walker(node, parameters, outerFont) {
   return parameters;
 }
 
-module.exports = function(html, callback) {
+module.exports = function(html, callback, matcher) {
+  if (matcher && typeof matcher === 'function') {
+    customMatcher = matcher;
```

Comment:
If you only set this module-scope variable when there *is* a matcher passed, then it will also be used on the next calls without a matcher passed.

I'd rather see this variable passed on to the walker so that it is not a module-scope